rentalPrice.js

```diff
@@ -1,40 +1,180 @@
-// age - age of driver
-// licence - number of full years person holds driving licence
-// clazz - class of the car from 1 (smallest) to 5 (largest) that person wishes to rent
-// acc - has s/he caused any accidents within last year
-// acc2 - has s/he participated (but not caused) in any accidents within last year
-// season - if it is high season or not
-function price(age, licence, clazz, acc, acc2, season) {
+// ageOfDriver - age - age of driver
+// yeasrOfDrivingLicence - licence - number of full years person holds driving licence
+// classOfCar - clazz - class of the car from 1 (smallest) to 5 (largest) that person wishes to rent
+// hasAccidents - acc - has s/he caused any accidents within last year
```

Comment:
hasAccidents - is not a good name, does not give idea if user caused or participated in an accident.